setup.py

```diff
@@ -37,7 +37,9 @@
 
     install_requires=[ 'biopython',
                        'matplotlib',
-		       'numpy',
+		               'numpy',
```

Comment:
would this whitespace make any problems? i think this is the only thing not perfect now.

Reply:
When I look at the code on the server (with Visual Studio Code) I don't see any white space 
![image](https://github.com/user-attachments/assets/48d269f4-664e-45ec-b42b-de98ec7ca78d)